.gitlab-ci.yml

```diff
@@ -249,6 +249,14 @@ before_script:
 # stage: deploy-gce-part1
   MOVED_TO_GROUP_VARS: "true"
 
+.ubuntu_contiv_sep_variables: &ubuntu_contiv_sep_variables
+# stage: deploy-gce-special
+  KUBE_NETWORK_PLUGIN: contiv
```

Comment:
We have moved to a new vars file system since this PR was opened. Can you delete this section and create a file tests/files/ubuntu-contiv-sep.yml with the following content?

```
# Instance settings
cloud_image_family: ubuntu-1604-lts
cloud_region: us-west1-a
mode: separate

# Deployment settings
kube_network_plugin: contiv
deploy_netchecker: true
kubedns_min_replicas: 1
cloud_provider: gce
```

Reply:
@lightcode: Can you take care of this, please?

@mattymo: Are any other changes needed? I'd like to have this merged soon.